rct229/rules/section15.py

```diff
@@ -212,7 +212,7 @@ def get_calc_vals(self, context, data=None):
 
             return {
                 "baseline_transformer_efficiency": context.baseline["efficiency"],
-                "required_baseline_transformer_efficiency": table_8_4_4_eff(
+                "required_baseline_transformer_efficiency": table_8_4_4_lookup(
```

Comment:
Needs to be modified since the return value for table_8_4_4_lookup() has changed.

Reply:
table_8_4_4_lookup() returns a dict of the form { "efficiency": --- }, so are you suggesting it to be changed to:

"efficiency": table_8_4_4_lookup(
                    phase=baseline_transformer_phase, kVA=baseline_transformer_kVA
                ),
 instead of - 
 "required_baseline_transformer_efficiency": table_8_4_4_lookup(
                    phase=baseline_transformer_phase, kVA=baseline_transformer_kVA
                ),
                
?

Reply:
No. I made a comment in your later code change.